File: src/main/java/org/tasks/PermissionUtil.java
package org.tasks;

import android.content.pm.PackageManager;

public abstract class PermissionUtil {
    public static boolean verifyPermissions(int[] grantResults) {
        if(grantResults.length < 1){
            return false;
        }

        for (int result : grantResults) {
            if (result != PackageManager.PERMISSION_GRANTED) {
                return false;
            }
        }
        return true;
    }
}

File: app/src/main/java/org/tasks/PermissionUtil.kt
package org.tasks

import android.content.pm.PackageManager.PERMISSION_GRANTED

object PermissionUtil {
    @JvmStatic
    fun verifyPermissions(grantResults: IntArray) =
            grantResults.isNotEmpty() && grantResults.all { it == PERMISSION_GRANTED }
}